dash-renderer/webpack.config.js

```diff
@@ -10,7 +10,7 @@ const defaults = {
         rules: [
             {
                 test: /\.js$/,
-                exclude: /node_modules/,
+                exclude: /node_modules\/(?!@plotly\/dash-component-plugins\/|uniqid\/|check-prop-types\/)/,
```

Comment:
Exclude all that can be excluded. In this case it means that `@plotly/dash-component-plugins`, `uniqid` and `check-props`types` need to be transpiled to the correct target b/c we're using the source or a dist with a less stringent target.